cypress/e2e/news/createReadDeleteNewsInAdvance.feature

```diff
@@ -0,0 +1,49 @@
+@stable_test
+@regression_test
+Feature:  News - To read a news on the respective dashboards
+
+    As a teacher I want to read the news shown on the dashboard so that I'm informed about the latest news
+
+    Scenario: User creates news with time options, reads them and deletes them
+        # as a pre-condition create users for author and reader
+        Given I am logged in as a '<news_reader>' at '<namespace>'
+        Given I am logged in as a '<news_author>' at '<namespace>'
+
+        # first user creates school news with different time options
+        When I go to news overview
+        And I click on add news button
+        Then I see news creation page
+        And I enter news title '<news_title>'
+        And I enter news description '<news_description>'
+        When I set news-visibility-start-date to '<news_day_from_today>' days at '<news_time>'
+        And I see time input field
+        And I click on save button
+        When I go to news overview
+        Then I do not see the unpublished news '<news_title>'
+        When I click on tab for unpublished news
+        Then I see the unpublished news '<news_title>'
+
+        # second user reads a school news on news overview page, then second user opens news and reads news detail page
```

Comment:
should be? -> _student reads a school news on news overview page, then second user opens news and reads news detail page_

Reply:
This test would be possible for different roles, so I want to keep it flexibel and not to define a special role.